tests/examplefiles/nim/example.nim.output

```diff
@@ -5546,176 +5542,155 @@
-'\n    '      Text
-'PCheckMenuItem' Name
-'('           Punctuation
-'caseInSensMenuItem' Name
-')'           Punctuation
-'.'           Punctuation
-'ItemSetActive' Name
-'('           Punctuation
-'True'        Keyword.Pseudo
-')'           Punctuation
-'\n  '        Text
-'of'          Keyword
-' '           Text
-'"'           Literal.String
-'style'       Literal.String
-'"'           Literal.String
-':'           Punctuation
-'\n    '      Text
-'PCheckMenuItem' Name
-'('           Punctuation
-'styleMenuItem' Name
-')'           Punctuation
-'.'           Punctuation
-'ItemSetActive' Name
-'('           Punctuation
-'True'        Keyword.Pseudo
-')'           Punctuation
-'\n  '        Text
-'of'          Keyword
-' '           Text
-'"'           Literal.String
-'regex'       Literal.String
-'"'           Literal.String
-':'           Punctuation
-'\n    '      Text
-'PCheckMenuItem' Name
-'('           Punctuation
-'regexMenuItem' Name
-')'           Punctuation
-'.'           Punctuation
-'ItemSetActive' Name
-'('           Punctuation
-'True'        Keyword.Pseudo
-')'           Punctuation
-'\n  '        Text
-'of'          Keyword
-' '           Text
-'"'           Literal.String
-'peg'         Literal.String
-'"'           Literal.String
-':'           Punctuation
-'\n    '      Text
-'PCheckMenuItem' Name
-'('           Punctuation
-'pegMenuItem' Name
-')'           Punctuation
-'.'           Punctuation
-'ItemSetActive' Name
-'('           Punctuation
-'True'        Keyword.Pseudo
-')'           Punctuation
-'\n\n  '      Text
-'extraMenu'   Name
-'.'           Punctuation
-'popup'       Name
-'('           Punctuation
-'nil'         Keyword.Pseudo
-','           Punctuation
-' '           Text
-'nil'         Keyword.Pseudo
-','           Punctuation
-' '           Text
-'nil'         Keyword.Pseudo
-','           Punctuation
-' '           Text
-'nil'         Keyword.Pseudo
-','           Punctuation
-' '           Text
-'0'           Literal.Number.Integer
-','           Punctuation
-' '           Text
-'get_current_event_time' Name
-'('           Punctuation
-')'           Punctuation
-')'           Punctuation
-'\n\n'        Text
-
-'# GUI Initialization' Comment
-'\n\n'        Text
+'"'           Name.Label
+'casesens'    Name.Label
+'":\n    '    Name.Label
+'PCheckMenuItem' Name.Label
+'('           Name.Label
+'caseSensMenuItem' Name.Label
+').'          Name.Label
+'ItemSetActive' Name.Label
+'('           Name.Label
+'True'        Name.Label
+')\n  '       Name.Label
+'of'          Name.Label
+' '           Text.Whitespace
+'"'           Name.Label
+'caseinsens'  Name.Label
+'":\n    '    Name.Label
+'PCheckMenuItem' Name.Label
+'('           Name.Label
+'caseInSensMenuItem' Name.Label
+').'          Name.Label
+'ItemSetActive' Name.Label
+'('           Name.Label
+'True'        Name.Label
+')\n  '       Name.Label
+'of'          Name.Label
+' '           Text.Whitespace
+'"'           Name.Label
```

Comment:
Is this intended? I don't know the language, but these "labels" seem large and they look like code to me ...